### PR TITLE
[thci] fix implementation of `setKeepAliveTimeOut`

### PR DESCRIPTION
**Pull request by edmont**
Thread 1.2 harness uses this method to set the child timeout in DUA-TC-10, DUA-TC-21 and DUA-TC-22.

### CODE REVIEW

File: tools/harness-thci/OpenThread.py [hunk 1]
```diff
@@ -1888,22 +1888,18 @@ def setNetworkIDTimeout(self, iNwkIDTimeOut):
...
-            iTimeOut: data poll period for sleepy end device
+            iTimeOut: child timeout for device
 
         Returns:
-            True: successful to set the data poll period for SED
-            False: fail to set the data poll period for SED
+            True: successful to set the child timeout for device
+            False: fail to set the child timeout for device
         """
         print('%s call setKeepAliveTimeOut' % self)
```
Comment (simonlingoogle) [inline tools/harness-thci/OpenThread.py:1900]: also remove this line since `@API` is already printing verbosely

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

File: tools/harness-thci/OpenThread.py (hunk 1 above)
Comment (edmont) [reply tools/harness-thci/OpenThread.py:1900]: Sure, will do.

**Comment by edmont** [comment]
@simonlingoogle what do you think we should do with the `OpenThread_WpanCtl.py` implementation?

**Comment by simonlingoogle** [comment]
> @simonlingoogle what do you think we should do with the `OpenThread_WpanCtl.py` implementation?

Should be fine to leave it unchanged since we are not using it in reference releases?

**Comment by edmont** [comment]
> Should be fine to leave it unchanged since we are not using it in reference releases?

Agree.

**Review by jwhui — APPROVED** [review]

**Comment by jwhui** [comment]
@edmont , can you help resolve conflicts?

**Comment by jwhui** [comment]
> @edmont , can you help resolve conflicts?

I went ahead and resolved :)

**Pull request merged**